Jenkinsfile

```diff
@@ -155,6 +155,7 @@ pipeline {
                             }
                         }
                         withMaven(maven: 'maven-latest', jdk: 'jdk8-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LINUX_MVN_RANDOM}') {
+                            sh 'mvn javadoc:aggregate -B -DskipStatic=true -DskipTests=true -nsu $DISABLE_DOWNLOAD_PROGRESS_OPTS'
```

Comment:
Does this run for PR builds?

Reply:
It might be better to move this inside the "Full Build" stage because it looks like this stage is only run on master branch